recipes/pyneval/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "pyneval" %}
+{% set version = "0.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 6707520a7f7e449d5503f9c81c280e4caa20ee789c13a1e852c765f70f6788ba
+
+build:
```

Comment:
I think this package can be `noarch: python`. See:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

If this is the case, you should change `python` to `python >=3.5` or `python =2.7|>=3.5` etc under both the `run` and `host` sections under `requirements`.